pkg/ddevapp/config.go

```diff
@@ -48,8 +48,8 @@ func init() {
 	if os.Getenv("DDEV_TEST_NO_BIND_MOUNTS") == "true" {
 		nodeps.NoBindMountsDefault = true
 	}
-	if os.Getenv("DDEV_TEST_USE_TRAEFIK") == "true" {
-		nodeps.UseTraefikDefault = true
+	if os.Getenv("DDEV_TEST_USE_TRADITIONAL_ROUTER") == "true" {
```

Comment:
Why not simply using DDEV_TEST_ROUTER and provide the correct value like done internally anyway? This would be more flexible to also be able to override with the Traefik router type for tests if needed at some point.

Reply:
I decided not to change this one, as the test being triggered here is a one-off. All other tests will use the default traefik.